stopwatch-MDAG/styles.css

```
@@ -0,0 +1,126 @@
+.btn-stop { background-color: #dc3545; }
+.btn-reset { background-color: #6c757d; }
+
+/* --- Nuevas clases de CodeRabbit --- */
+
+/* Clase para ocultar elementos visualmente pero mantenerlos para lectores de pantalla */
+.visually-hidden {
+    position: absolute;
+    width: 1px; height: 1px;
+    padding: 0; margin: -1px;
+    overflow: hidden;
+    clip: rect(0,0,0,0);
```

Stylelint (17.9.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)